tests/install_browsers.sh

```diff
@@ -2,6 +2,7 @@
 
 export DEBIAN_FRONTEND="noninteractive"
 
+TARGET=/usr/local/bin
```

Comment:
Can we name it `WEBDRIVER_BIN` instead of `TARGET`?

Reply:
okay